tests/testdata/python3/data/recursion.py

```diff
@@ -1,3 +1,3 @@
-""" For issue #25 """
-class Base(object):
+""" For issue #25 """
+class Base(object):
     pass
```

Comment:
There is a missing line missing here though @Pierre-Sassoulas 😄 

But no reason to add it now!